apps/expo/src/notifications/registerNotifications.tsx

```diff
@@ -0,0 +1,39 @@
+import { Platform } from "react-native";
+import Constants from "expo-constants";
+import * as Device from "expo-device";
+import * as Notifications from "expo-notifications";
+
+async function registerForPushNotificationsAsync() {
+  let token;
+
+  if (Platform.OS === "android") {
+    void Notifications.setNotificationChannelAsync("default", {
+      name: "default",
+      importance: Notifications.AndroidImportance.MAX,
+      vibrationPattern: [0, 250, 250, 250],
+      lightColor: "#FF231F7C",
+    });
+  }
```

Comment:
is there another check required for iOS, or is this only needed for android?

Reply:
The Expo docs only showed a check for android. It works fine on iOS so I assume it's fine